### PR TITLE
docs(ja): translate deployment/deployment-azure-portal

### PR DESCRIPTION
**Pull request by Katakuranatsumi**
Please review this PullRequest.
ref vuejs-jp#251

### CODE REVIEW

File: content/ja/guides/deployment/deployment-azure-portal.md [hunk 1]
```diff
@@ -133,16 +133,16 @@ For Azure Portal you will need a `web.config` file. If not supplied, it will cre
...
-        * watchedFiles: semi-colon separated list of files that will be watched for changes to restart the server
-        * node_env: will be propagated to node as NODE_ENV environment variable
-        * debuggingEnabled - controls whether the built-in debugger is enabled
+      以下のオプションを使用すれば、IIS 内で Node がホストされる方法を制御することができます:
+        * watchedFiles: サーバーを再起動する時の変更を監視する、セミコロンで区切られたリストのファイル
+        * node_env: 環境変数 NODE_ENV として node に伝わります
+        * debuggingEnabled - 組み込みのデバッガを有効にするかどうかを制御します
 
-      See https://github.com/tjanczuk/iisnode/blob/master/src/samples/configuration/web.config for a full list of options
+      オプションの全リストは https://github.com/tjanczuk/iisnode/blob/master/src/samples/configuration/web.config で確認してください
```
Comment (inouetakuya) [inline content/ja/guides/deployment/deployment-azure-portal.md:145]: Your translation is natural and easy to read 👍

**Review by inouetakuya — APPROVED** [review]
Thank you for your contribution :+1:

File: content/ja/guides/deployment/deployment-azure-portal.md (hunk 1 above)
Comment (Katakuranatsumi) [reply content/ja/guides/deployment/deployment-azure-portal.md:145]: 良かったです、ありがとうございます😄

**Comment by Katakuranatsumi** [comment]
ご確認ありがとうございました🙏！

**Pull request merged**